tests/subsys/bluetooth/fast_pair/storage/account_key_storage/src/test_corrupted_data.c

```diff
@@ -212,42 +218,69 @@ ZTEST(suite_fast_pair_storage_corrupted, test_inconsistent_key_id)
 
 ZTEST(suite_fast_pair_storage_corrupted, test_drop_first_key)
 {
-	store_key(next_account_key_id(ACCOUNT_KEY_MIN_ID));
+	if (ACCOUNT_KEY_CNT < 2) {
+		store_key(ACCOUNT_KEY_MIN_ID + 1, false);
+	} else {
+		store_key(next_account_key_id(ACCOUNT_KEY_MIN_ID), false);
+	}
 
 	initialization_error_validate();
 }
 
 ZTEST(suite_fast_pair_storage_corrupted, test_drop_key)
 {
+	if ((ACCOUNT_KEY_CNT < 2) &&
```

Comment:
Since you added the build assertion, maybe we could simplify this condition to `if (ACCOUNT_KEY_CNT < 2)` ? (also for `test_drop_keys`)